docs/logging-format.md

```diff
@@ -228,6 +228,36 @@ and TLS probe points.
 | `tls::key_exchange_algorithm` | uint16     | Key exchange mode: ECDHE(0), DHE(1), PSK(2), ECDHE-PSK(3), DHE-PSK(4)                            |
 | `tls::group`                  | uint16     | Groups used in the handshake (as in IANA [registry][iana-tls-supported-groups])                  |
 
+##### SSH context names
+
+| name                   | description                 |
+|------------------------|-----------------------------|
+| `ssh::handshake_client`| SSH handshake for client    |
+| `ssh::handshake_server`| SSH handshake for server    |
+| `ssh::client_key_sign` | SSH client key signature    |
+| `ssh::server_key_sign` | SSH server key signature    |
```

Comment:
Do these context cover both signing and verification?

Reply:
Yes, will adjust